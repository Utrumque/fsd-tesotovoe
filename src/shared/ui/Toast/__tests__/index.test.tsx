import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';
import ToastContainer from '../index';

describe('ToastContainer', () => {
  it('рендерится без ошибок', () => {
    expect(() => render(<ToastContainer />)).not.toThrow();
  });
});
