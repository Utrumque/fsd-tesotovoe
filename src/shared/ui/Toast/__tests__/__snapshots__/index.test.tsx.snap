// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ToastContainer рендерит ReactToastContainer 1`] = `
<div>
  <section
    aria-atomic="false"
    aria-label="Notifications Alt+T"
    aria-live="polite"
    aria-relevant="additions text"
    class="Toastify"
  />
</div>
`;
